src/data.js

```diff
@@ -82,7 +82,7 @@ QueryBuilder.prototype._validateValue = function(rule, value) {
                 break;
 
             default:
-                tempValue = $.isArray(value[i]) ? value[i] : [value[i]];
+                tempValue = value.toString().split(',')[i];
```

Comment:
Are you sure about this? Cause the below code assumes that `tempValue` is an array but it won't be anymore, right?